Documentation/Upgrade/rook-upgrade.md

```diff
@@ -46,6 +46,12 @@ those releases.
     official releases. Builds from the master branch can have functionality changed or removed at any
     time without compatibility support and without prior notice.
 
+## Breaking changes in v1.13
+
+* The minimum supported version of Kubernetes is v1.23.
+* Support for the admission controller/webhooks has been removed. If admission controller/webhooks is enabled, disable by changing
```

Comment:
Could you manually validate the behavior if they have the admission controller and forget to enable it before upgrading? Will the artifacts leftover from the webhook cause any issues?

Reply:
I tested locally, initially had webhook created and then went this `add-cel` branch applied the crds.yaml. No issues were found also I created a pool and created it successfully.

Webhooks leftover certificates should not cause any issue with CEL but it's better to remove the webhooks

Reply:
Great, thanks for the confirmation that it won't be breaking if they miss disabling it!